packages/wmr/test/test-helpers.js

```diff
@@ -50,6 +51,18 @@ export async function teardown(env) {
 export async function loadFixture(name, env) {
 	const fixture = path.join(__dirname, 'fixtures', name);
 	await ncp(fixture, env.tmp.path);
+	// For as long as the main "wmr" entry is self contained we
```

Comment:
Idea: what about running `npm link` here instead?

Reply:
I'd prefer to avoid that as `npm link` relies on setting up symlinks globally. It would introduce an external point of failure in our test suite.

Reply:
This gave me an idea! We can just leverage symlinks without npm via `fs.symlink`. It seems to work locally 👍

Reply:
Tests are green on CI too 🎉

Reply:
It's too error prone, much easier to just copy that one file into a custom `node_modules` dir. Keeps the test fixture isolated too.